bundle/manifests/datasciencecluster.opendatahub.io_datascienceclusters.yaml

```diff
@@ -245,6 +245,17 @@ spec:
                         - Removed
                         pattern: ^(Managed|Unmanaged|Force|Removed)$
                         type: string
+                      nim:
+                        description: Configures and enables Nvidida's NIM integration
```

Comment:
Correct Typo  Nvidida's NIM -> NVIDIA NIM

Reply:
We just named it generically. Do you want me to change? @swati-kale

Reply:
I see what you're saying now I'll update.